docs/source/synthetic_population.rst

```diff
@@ -0,0 +1,87 @@
+female_13,TAZ,persons,100000,POPF13,(persons.AGEP >= 55) & (persons.AGEP < 60) & (persons.SEX == 2)
+female_14,TAZ,persons,100000,POPF14,(persons.AGEP >= 60) & (persons.AGEP < 65) & (persons.SEX == 2)
+female_15,TAZ,persons,100000,POPF15,(persons.AGEP >= 65) & (persons.AGEP < 70) & (persons.SEX == 2)
+female_16,TAZ,persons,100000,POPF16,(persons.AGEP >= 70) & (persons.AGEP < 75) & (persons.SEX == 2)
+female_17,TAZ,persons,100000,POPF17,(persons.AGEP >= 75) & (persons.AGEP < 80) & (persons.SEX == 2)
+female_18,TAZ,persons,100000,POPF18,(persons.AGEP >= 80) & (persons.SEX == 2)
+
+For each sex and age structure, we create a control variable, considering the geography level of TAZ. For both women and men, we have 18 variables, named POPF and POPM, for female and male inhabitants. For TAZs we also consider the expected number of households. In the following section, we discuss the premisses of the model, and explain how the expected number of households was calculated. Finally, we have the controls by REGION, an upper level. For regional control, we consider the expected number of households of different sizes.
+
+Tradesman model premisses
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PopulationSim library requires a control file to run its processes. In the control file, one can specify and check the requirements for the synthetic population. The only mandatory requirement for a PopulationSim is generating the right number of households in each geography and this specification is made at the lowest level. The lowest level in Tradesman is the TAZs (Traffic Assessment Zones), however, the data necessary to estimate the total households in the upper level. 
+
+To fulfill the TAZ requirement, we estimated the total households based on the United Nations `Population Division <https://www.un.org/development/desa/pd/data/household-size-and-composition>`_ information about household composition. As there is no data available for all countries or autonomous regions, for countries that are not present in the sample, we considered their household informations to be similar to their closest neighbors. As there are many data sources available in the Population Division sheet, we kept the most recent data for each country, as long as it had information about the total number of households.
+
+The total number of households in each TAZ is a sum of all households with different sizes. To match the information provided by the UN, we consider households with 1 person (HHBASE1), 2 to 3 persons (HHBASE2), 4 to 5 persons (HHBASE4), and with more than 6 persons (HHBASE6). Hence, we sum all population by sex and age in the zone, divide by the average number of persons in the household, and multiply by the percentage of households of each size expected for the country. Finally, we sum the integer expected values for each household size to obtain the mandatory expected number of households in the TAZ (HHBASE).
```

Comment:
What number did you use for the average size of families with 6 or more people?  It is necessarily larger than 6...